### ISSUE TITLE
Feature Request- Add in gnomAD SV call set to prior

### ISSUE THREAD

**Issue by jjfarrell** [bioinformatics-centre/BayesTyper#27]
The prior vcf that is distributed BayesTyper has a very comprehensive list of SVs. Now there is a recently released SV vcf from gnomAD would be great to add to the present prior list: gnomadSV v2.1 vcf is found at   https://gnomad.broadinstitute.org/downloads#v2-structural-variants.

**Comment by jonassibbesen** [comment]
Good idea! Thank you for the suggestion. As previously mentioned I do not have much time to work on BayesTyper currently. I have labeled it as an enhancement and will keep the issue open as a reminder. In the meantime, you can maybe create it yourself by merging the prior with the gnomAD callset using `bayesTyperTools combine`. Remember to make sure that the contigs appear in the same order in the header and for the variant entries in the two files.
